_sass/_footer.scss

```diff
@@ -29,8 +35,8 @@
   }
 }
 
-@include media-breakpoint-down('md') {
-  .footer-item {
-    display: block;
+@include media-breakpoint-up('xl') {
+  .footer {
+    margin-top: -2.5rem;
```

Comment:
Actually I just added 

```js
@include media-breakpoint-up('xl') {
  .footer {
    margin-top: -2.5rem;
  }
}
```

The rest was because the breakpoints were not in the right order - that's why git triggered that many changes.